client/src/components/MultiJurisdictionAudit/RoundManagement/generateSheets.ts

```diff
@@ -39,13 +39,15 @@ export const downloadLabels = async (
     })
     labels.autoPrint()
     labels.save(`Round ${roundNum} Labels.pdf`)
+    return labels.output()
```

Comment:
maybe leave a note that you're returning the value for the test - I could see my future self finding this code and being confused

Reply:
@jonahkagan Oh yes, forgot to do that. Thanks!